### PR TITLE
Making consistent with our enrichments

### PR DESCRIPTION
**Pull request by khumphrey**
I am thinking that we need to REQUIRE react-redux for replays, so I did as much, and got them started with Provider. Thoughts?

@coreygreen1108 I know you mentioned wanting more detail than the original. I cleaned up the README (what is clean to me obv) and tried to keep a lot of detail. What do you think?

@Kmacpher @cassiozen We were the ones talking about replay stuff, so I thought I would include you for review. Thoughts?

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-const app = require('./server');
```
Comment (coreygreen1108) [inline main.js]: why are we getting rid of this?

File: readme.md [hunk 2]
```diff
@@ -1,96 +1,138 @@
...
+ Design a website for aircraft fanatics and create a RESTful API that allows you to manage these aircrafts! 
+
+### The tools
+
+Use at least Sequelize, Express, React, Redux, and React-Redux when creating this app. You can incorporate any additional libraries or tools you wish.
+
+### DB Design
+
+#### Aircrafts
+  * have profile info including:
```
Comment (coreygreen1108) [inline readme.md]: still think we may want to consider keeping examples.

File: client/components/Root.jsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
+import React, { Component } from 'react';
+
+export default class WinterJokes extends Component {
```
Comment (coreygreen1108) [inline client/components/Root.jsx]: I think this as a root is confusing given the premise of the project.

File: server/index.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
 const express = require('express');
-const app = express();
 const path = require('path');
 const volleyball = require('volleyball');
 const bodyParser = require('body-parser');
+const db = require('../db/_db')
+const port = 1337;
```
Comment (coreygreen1108) [inline server/index.js]: Please capitalize if we're going to be providing them a constant.

File: server/api/index.js [hunk 5]
```diff
@@ -1,6 +1,9 @@
-const router = require('express').Router(); 
+const apiRouter = require('express').Router(); 
+const db = require('../db')
 
-router.use('/aircraft', require('./aircraft')); 
-router.use('/countries', require('./country')); 
+// If you aren't getting to this object, but rather the index.html (something with a joke) your path is wrong.
+	// I know this because we automatically send index.html for all requests that don't make sense in our backend.
+	// Ideally you would have something to handle this, so if you have time try that out!
+api.get('/hello', (req, res) => res.send({hello: 'world'}))
```
Comment (coreygreen1108) [inline server/api/index.js]: this is going to throw an error?

File: server/api/index.js [hunk 6]
```diff
@@ -1,6 +1,9 @@
-const router = require('express').Router(); 
+const apiRouter = require('express').Router(); 
+const db = require('../db')
 
-router.use('/aircraft', require('./aircraft')); 
-router.use('/countries', require('./country')); 
+// If you aren't getting to this object, but rather the index.html (something with a joke) your path is wrong.
```
Comment (coreygreen1108) [inline server/api/index.js]: Liked the idea of requiring them to break their routes into smaller pieces.

File: main.js (hunk 1 above)
Comment (khumphrey) [reply main.js]: It doesn't make sense. We should just start with the server I would think. See the server/index.js file

File: readme.md (hunk 2 above)
Comment (khumphrey) [reply readme.md]: I like that we give them the room to be creative and own it and have fun with it!

File: client/components/Root.jsx (hunk 3 above)
Comment (khumphrey) [reply client/components/Root.jsx]: Yeah it is just example code. But good points that maybe I should add some comments to explain that

File: server/api/index.js (hunk 5 above)
Comment (khumphrey) [reply server/api/index.js]: Noice catch :D

File: server/api/index.js (hunk 6 above)
Comment (khumphrey) [reply server/api/index.js]: They should! And they should show they know how to do this on their own. This is something we are grading on

File: server/index.js (hunk 4 above)
Comment (khumphrey) [reply server/index.js]: Good point!

**Comment by khumphrey** [comment]
It worked so well, so I am going to try again! @omriBernstein good for a review??? 🙏

**Comment by omriBernstein** [comment]
Haha sure, but should I wait for Corey's changes to be resolved? @khumphrey

**Comment by khumphrey** [comment]
@omriBernstein I think that you should give your opinion to those comment!

File: client/components/Root.jsx (hunk 3 above)
Comment (omriBernstein) [reply client/components/Root.jsx]: +1 think it's worthwhile to add an explanation somewhere that this is just the example root we start them off with

Comment (omriBernstein) [reply client/components/Root.jsx]: Oh and I see, @coreygreen1108 you're also saying that it'll be confusing given the whole "you're building an app for aircraft fanatics". I could see that, but I also don't think it matters much what example code we start them off with for their React app root.

File: main.js (hunk 1 above)
Comment (omriBernstein) [reply main.js]: Yeah either way seems reasonable. And all things being equal, I tend to think it's simpler to have fewer files.

File: readme.md (hunk 2 above)
Comment (omriBernstein) [reply readme.md]: Where are/were these examples?

File: server/api/index.js (hunk 6 above)
Comment (omriBernstein) [reply server/api/index.js]: +1 I think we should rely on them to do this on their own

**Comment by omriBernstein** [comment]
Done...I think

File: readme.md (hunk 2 above)
Comment (khumphrey) [reply readme.md]: In the original @omriBernstein. But it was just Lockeed Martin as an example `make`

Comment (omriBernstein) [reply readme.md]: Oh oh I see. Yeah I think that's totally worth providing an example—at least, I can imagine that I myself would be confused by what "make" would mean, or even "model". I'm not very vehicle-savvy.

Comment (omriBernstein) [reply readme.md]: So I vote to keep the example

File: main.js (hunk 1 above)
Comment (coreygreen1108) [reply main.js]: I guess i can understand the fewer files thing but most larger applications start with a main.js or app.js root. Just didn't feel the need to just rid this

File: server/api/index.js (hunk 6 above)
Comment (coreygreen1108) [reply server/api/index.js]: sure

**Pull request merged**